Portfolio: desktop version

david63011 opened the pull request:
here is the version of my desktop site built with media queries as well as HTML and CSS

Qoosim reviewed (changes requested):
# **Status: Required Changes** :recycle: 

Hi @david63011, 

Good job so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Top Highlights

-  No linter errors. :heavy_check_mark: 
-   Adherence to Github Flow. :heavy_check_mark: 
-  Good commit messages. :heavy_check_mark: 

![](https://user-images.githubusercontent.com/20554583/186796336-bf5a29c8-3a54-4d16-be46-376571590867.gif)
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Good job for pushing it this far! :clap: . However, there are few things you still need to work on before your project can get approval.
     -  First, the margin right and left of your portfolio is not consistent. All the contents of your page is cut off on the right of your page. Kindly look into that and fix it. 

### Figma Design

![figma-design-2](https://user-images.githubusercontent.com/26597432/186950782-d2ae09d8-1367-4fa4-97f4-e060ec621031.png)


### Your Design

![your-design-3](https://user-images.githubusercontent.com/26597432/186950908-39c90737-db1a-4fd1-bf49-5050918ec4c6.png)

- [ ] Also, try to look into your placeholder texts in your input field, they are too closed to the left wall of your input field. Kindly give your placeholder texts some margin or padding left to move them a bit to the right.

### Figma Design

![figma-design-1](https://user-images.githubusercontent.com/26597432/186951597-9d5ee78e-a3d1-432a-9cca-bb8b62499e05.png)


### Your Design

![your-design-1](https://user-images.githubusercontent.com/26597432/186951692-81c2a2a5-02a7-4919-858a-21760a2ea5e2.png)

- [ ] Try to follow the Figma Design as close as possible. This includes, font, font-size, font-family, font-weight, color, margin, padding, border, among others. Doing this would make our portfolio to look standard and professional.

### Figma Design

![figma-btn](https://user-images.githubusercontent.com/26597432/186952488-315c9d98-714a-44c6-802e-18b0a66bfc96.png)


### Your Design

![your-btn](https://user-images.githubusercontent.com/26597432/186952528-d062ab87-8fa4-4931-a0fb-ab9e2a7ab76a.png)

-  Kindly remove the border from your `Get in touch` button.

- [ ]  I would also like to recommend you to look into the margin left and right of your list of projects section and give it equal margin left and right.

### Figma Design

![figma-works](https://user-images.githubusercontent.com/26597432/186953378-609bdb56-d4e2-4617-9ee2-114174fd3e1b.png)


### Your Design

![your-design-5](https://user-images.githubusercontent.com/26597432/186953492-b098a23f-94ae-4019-bd95-b49d7e5f2da6.png)

- [ ] Kindly add more information to README file and make it well organized. It does not look too professional with the current outlook. you can make it look better by following this [template](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse

david63011 commented:
ok I fixed the margin problem on the right,  I fixed the placeholder text, I fixed the buttons, I fixed the margin on the project section, and I improved my readme

mgmediaweb reviewed (changes requested):
Hi @david63011, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![work2](https://user-images.githubusercontent.com/41912912/187014976-4c409952-705f-442b-9195-fa12727d0526.gif)

 ## Highlights 💯 

- I appreciate the effort you put into your work ✔️ 

 ## Required Changes ♻️

- kindly advise you to follow the previous reviewer's advice.
- Kindly, it has quite a few differences from the original Figma design, I recommend you review your site thoroughly and compare fonts, images, colors, spaces between elements, and padding.

_Check the comments under the review._

## Optional suggestions

[OPTIONAL] Kindly, the initial file of your web page should always be index.html (not index2.html) I suggest you change it

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

david63011 commented:
hi @mgmediaweb I am happy to make changes could you please be a little more specific i only have one code review left

rukundoeric reviewed (changes requested):
Hello @david63011 👋,

_This is an additional review as a result of the second opinion request._

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._

## Required Changes ♻️
- [ ] As mentioned by the previous reviewer, there still a lot you have to do to make your app look more like the Figma design. 
  - The app does not respond well either in small and big screen.  You can learn more about css responsiveness [Here](https://www.w3schools.com/css/css_rwd_intro.asp)
  - There are so much overlapping, check out the screenshots below
 
| Original      | Yours |
  | ----------- | ----------- |
| ![Screenshot(2)](https://user-images.githubusercontent.com/45217364/187035093-e0aac7e7-3003-4f7d-9c39-7953c7d22473.png) | ![Screenshot (1)](https://user-images.githubusercontent.com/45217364/187035176-9c49207d-3716-4ab6-8c69-028e29311848.png) |

Kindly fix it.


_Check the comments under the review._


Keep rocking 🚀!

david63011 commented:
Ok I remade my entire middle section to make it more responsiveness  I hope that is enough

david63011 commented:
as well as change other projects with the desktop page

ElsonOtake commented on style2.css:
```diff
@@ -33,6 +116,21 @@ body {
+  margin-left: 260px;
+}
+
+.paragraph2 {
```
- [ ] Please correct the overlapping of text as required by the previous reviewer. Stick to the Figma design is a project requirement.
Your portfolio:
<img width="585" alt="Screen Shot 2022-08-27 at 7 51 40 PM" src="https://user-images.githubusercontent.com/47673311/187050844-aed99706-3d74-4db0-87f0-34d9de3a819b.png">

Figma template:
<img width="694" alt="Screen Shot 2022-08-27 at 7 51 59 PM" src="https://user-images.githubusercontent.com/47673311/187050855-ca88e5fc-c8c5-4758-a504-f8d3dd140444.png">

BenMukebo replied:
- [ ] Kindly address this 👍🏻

ElsonOtake commented on style2.css:
```diff
@@ -1157,15 +1015,37 @@ textarea {
     width: 369px;
   }
 
+  .type-of-cards {
```
- [ ] Please correct the overlapping as requested by the previous reviewer. Stick to the Figma template is a project requirement.
Your portfolio:
<img width="488" alt="Screen Shot 2022-08-27 at 7 53 34 PM" src="https://user-images.githubusercontent.com/47673311/187050898-2080fbab-bb4d-429e-a79a-ffd0368878fe.png">

Figma template:
<img width="706" alt="Screen Shot 2022-08-27 at 7 54 53 PM" src="https://user-images.githubusercontent.com/47673311/187050905-70622fb7-37f6-484d-9023-223b685d33ce.png">

david63011 replied:
@ElsonOtake ok i have been trying all day i even rebuilt the whole site do you have any advise on how to do it

david63011 replied:
and now it is responsive I dont want next week to only resize the page

BenMukebo replied:
- [ ] Kindly address this 👍🏻

ElsonOtake reviewed (changes requested):
STATUS: CHANGES REQUIRED 🟡 

Hi @david63011, 

Great work on making the changes requested by a previous reviewer 👏 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 🥲 🥲 

![woohoo-youre-almost](https://user-images.githubusercontent.com/77038610/154921145-1f7bb8f6-c843-4ba5-a534-6b840c600558.jpeg)

 ## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. 👍 👍 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

david63011 commented:
ok I fixed the overlapping of text,  I created more empty spaces on both sides (left and right) of your work sections, and i improved my readme

BenMukebo commented on index2.html:
```diff
@@ -292,56 +304,56 @@ <h6 class="textline7">in touch</h6>
-
-          <button type="submit" class="touch-btn">Get in touch</button>
+          <div class="input-box-box">
+            <ul class="form-info">
+              <li>
+                <input
+                  type="text"
+                  maxlength="30"
+                  name="user_name"
+                  class="name-text-box"
+                  id="full-name"
+                  placeholder="Full Name"
+                  required=""
+                />
+              </li>
+
+              <li>
+                <input
+                  type="email"
+                  name="user_email"
+                  class="name-last-text-box"
+                  id="email"
+                  placeholder="Email Address"
+                  required=""
+                />
+              </li>
+              <li>
```
> * [ ]  I think it's better to remove the border of this button as requested by the previous reviewer. Stick to the Figma template is a project requirement.
>   Your portfolio:
> 
> <img alt="Screen Shot 2022-08-27 at 7 09 47 PM" width="779" src="https://user-images.githubusercontent.com/47673311/187049669-378439fb-8be7-4510-bd77-c67760e33022.png">
> 
> Figma template: <img alt="Screen Shot 2022-08-27 at 7 10 39 PM" width="705" src="https://user-images.githubusercontent.com/47673311/187049678-a5a7140a-fd65-48d8-aa8d-50cdd992cd80.png">

- [ ] Kindly address this 👍🏻

BenMukebo commented on style2.css:
```diff
@@ -1157,15 +1015,37 @@ textarea {
+    border-radius: 5px;
+  }
+
   .grid-containers {
```
- [ ] Great job so far, However you should make sure to center your work sections on the center of the page as requested by the previous reviewer, the margin-left should be legal to the margin-right

BenMukebo commented on README.md:
```diff
@@ -1,10 +1,28 @@
+
+The first step in setting up this project in setting up your lintors. After that you can go to Figma and find a template and then use your HTML and CSS
+to design your site. Good luck!
+
```
- [ ] Kindly address this 👍🏻

BenMukebo reviewed (changes requested):
Hi @david63011,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight :clap: 
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️


 ## Required Changes ♻️
_Check the comments under the review._

- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Please Instead of disabling `eslint`, it will be nice to fix all linter errors manually.

You have submitted a project, that Linters check are not running in the GitHub/action
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please Tag me on (@BenMukebo) when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
